website/content/docs/providers.md

```diff
@@ -34,61 +34,72 @@ cd virtual-kubelet
 mkdir providers/my-provider
 ```
 
-In that created directory, implement the [`Provider`](https://godoc.org/github.com/virtual-kubelet/virtual-kubelet/providers#Provider) interface in [Go](https://golang.org).
+In that created directory, implement the [`Provider`](https://godoc.org/github.com/virtual-kubelet/virtual-kubelet/cmd/virtual-kubelet/internal/provider#Provider) interface in [Go](https://golang.org).
```

Comment:
What about the `PodLifecycleHandler`?

Reply:
Agree this should be `PodLifecycleHandler` since users of this package do not have access to `internal`